packages/web/src/utils/prerender.utils.tsx

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from 'react'
```

Comment:
I'm usually not a fan of creating a `utils` directory, since it ends up being a large collection of unrelated things.

Maybe this file is OK to just be `src/prerender`?

Reply:
Sure, no problem. Will do

Reply:
I'm wondering if a bunch of these couldn't just live in the prerender package instead?

Reply:
The reason I moved it into web src is because if I expose them from the prereder package, it tries to load index.tsx (which is meant to be server only). Same issue as the ApolloProvider Tobbe was facing

Reply:
Ok, here's a solution for solving that:

1. create a `./packages/browser|utils|runtime.js` file. Make sure it's added to the `files` part of the pre-renders `package.json`.
2. `./package/browser|utils|runtime.js` should be common-js (no transpiling) and just re-exports `./dist/utils.js`
3. then a user can do this: `import { useIsBrowser } from '@redwoodjs/prerender/runtime'

Reply:
Ping me if you need some help, but a little something like this early on is easier to do than trying to undo it later.